login/src/index.html

```diff
@@ -0,0 +1,119 @@
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this license. If not, see
+
+-->
+
+<!DOCTYPE html>
+<html manifest="manifest.appcache">
+
+<head>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="description" content="Elastic Path&trade; Visual Cortex - App Framework">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="theme-color" content="#5bc0de" />
+
+  <meta name='viewport' content='viewport-fit=cover, user-scalable=no, width=device-width, height=device-height, initial-scale=1.0'>
+
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+  <script
+    src="https://code.jquery.com/jquery-3.3.1.min.js"
+    integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+    crossorigin="anonymous"></script>
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,600" rel="stylesheet">
+
+  <title>Elastic Path&trade; Storefront Reference Application</title>
+
+  <script type="text/javascript">try { Typekit.load(); } catch (e) { }</script>
+
+</head>
+
+<body>
+
+  <!-- Viewport -->
+  <div class="viewport ui-container" data-region="viewPortRegion"></div>
+
+  <div id="root"></div>
+
+  <noscript>
+    <style type="text/css">
+      body {
+        overflow: hidden;
+      }
+
+      .noscript-overlay {
+        background-color: white;
+        bottom: 0;
+        left: 0;
+        position: absolute;
+        top: 0;
+        right: 0;
+        z-index: 2000;
+      }
+
+      .noscript-contain {
+        height: 200px;
+        left: 50%;
+        margin-left: -300px;
+        margin-top: -100px;
+        position: absolute;
+        top: 50%;
+        width: 600px;
+      }
+
+      .ep-style-wrapper .regular,
+      .ep-style-wrapper p {
+        font-weight: 400;
+      }
+
+      .ep-style-wrapper .font-xl {
+        font-size: 32px;
+        line-height: 36px;
+      }
+    </style>
+    <div class="noscript-overlay">
+      <div class="noscript-contain ep-style-wrapper">
+        <span class="font-xl regular colour-dark-blue margin-s-bottom block" style="display: block;">This website requires
+          JavaScript
+        </span>
+        <p class="font-xs margin-s-bottom">The features of this website require JavasScript to function. Please check your
+          browser settings and turn on JavaScript.</p>
+        <p class="font-xs">If you can't find the setting, try searching the web for your browser name (Chrome, Firefox, etc.)
+          and "turning on JavaScript".
+        </p>
+      </div>
+    </div>
+  </noscript>
+  <script src="https://www.gstatic.com/firebasejs/5.6.0/firebase.js"></script>
+  <script>
+  // Initialize Firebase
+    var config = {
+      apiKey: "AIzaSyBBKTbw3-gS214A_o0MAq7qQSQf_Mz5x0U",
```

Comment:
Are these your personal api credentials? If so I suggest replacing anything that is sensitive to you with some dummy values, or leave them empty similar to how we set our `gaTrackingId` here:
https://github.com/elasticpath/react-pwa-reference-storefront/blob/master/src/ep.config.json

Reply:
Fixed